File: MegaHerdt.Models/Models/ReparationClaimAnswer.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MegaHerdt.Models.Models
{
    public class ReparationClaimAnswer
    {
        [Key]
        public int Id { get; set; }
        [ForeignKey("ReparationClaim")]
        public int ReparationClaimId { get; set; }
        public string Answer { get; set; }
        public string EmployeeUserName { get; set; }
        public ReparationClaim ReparationClaim { get; set; }
    }
}

File: MegaHerdt.FrontEnd/MegaHerdt/src/app/models/ReparationClaims/ReparationClaimAnswer.ts
export class ReparationClaimAnswer{
    id: number;
    reparationClaimId: number;
    answer: string;
    employeeUserName: string;

    constructor(){
        this.id = 0;
        this.reparationClaimId = 0;
        this.answer = '';
        this.employeeUserName = '';
    }
}